math/euclidean_algorithm_extended.c

```diff
@@ -0,0 +1,148 @@
+/**
```

Comment:
Please remove this file from the PR (it appears you already have a PR for this).